packages/hatchify-core/vite.config.ts

```diff
@@ -0,0 +1,13 @@
+// todo: remote vite from hathciy-core w/o breaking frontend packages
```

Comment:
@roy-coder the plan is to get rid of this again - unfortunately required until we can resolve the builds for fe&be without vite